src/systems/systems/src/shared/arinc429.rs

```diff
@@ -40,6 +40,35 @@ impl<T: Copy> Arinc429Word<T> {
         self.ssm == SignStatus::NormalOperation
     }
 }
+impl Arinc429Word<f32> {
+    pub fn set_bit(&mut self, bit: u32, value: bool) {
```

Comment:
**suggestion**: I think we will not exceed 256 bits within set_bit. Therefore this can be a `u8`.
```suggestion
    pub fn set_bit(&mut self, bit: u8, value: bool) {
```

**suggestion**: Add `debug_assert!` to limit the valid usage to between 1 and 32 (or 0 and 31, depending on the standard).

Reply:
Implemented the first suggestion, but why only `debug_assert`?

Reply:
Alright, added the `debug_assert` for the bit range that is possible with the ARINC 429 data word structure (bits 11 to 29).